src/Features/CSharp/Portable/CodeStyle/TypeStyle/TypeStyle.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeStyle.TypeStyle
+{
+    [Flags]
+    internal enum TypeStyle
```

Comment:
It can sometimes be problematic to have a type and namespace with the same name.  I would avoid.